README.md

```diff
@@ -177,6 +177,9 @@ It is highly recommended you set the following environment values before startin
 | BACKUP_ENABLED | Enables automatic backups | true | true/false |
 | DELETE_OLD_BACKUPS | Delete backups after a certain number of days                                                                                                                                                       | false          | true/false                                                                                                 |
 | OLD_BACKUP_DAYS    | How many days to keep backups                                                                                                                                                                       | 30             | any positive integer                                                                                       |
+| AUTO_UPDATE_CRON_EXPRESSION  | Setting affects frequency of automatic updates. | 0 0 \* \* \* | Needs a Cron-Expression - See [Configuring Automatic Backups with Cron](#configuring-automatic-backups-with-cron) |
+| AUTO_UPDATE_ENABLED | Enables automatic updates | true | true/false |
```

Comment:
`AUTO_UPDATE_ENABLED` is set to false in the dockerfile.
```suggestion
| AUTO_UPDATE_ENABLED | Enables automatic updates | false | true/false |
```

Reply:
Yes first wanted to make it default true but since it's not yet tested enough false is better. Forgot to change that.